compiler/test/dotty/Properties.scala

```diff
@@ -8,8 +8,8 @@ import java.nio.file._
 object Properties {
 
   /** If property is unset or "TRUE" we consider it `true` */
-  private def propIsNullOrTrue(prop: String): Boolean = {
-    val prop = System.getProperty("dotty.tests.interactive")
+  private def propIsNullOrTrue(name: String): Boolean = {
+    val prop = System.getProperty(name)
     prop == null || prop == "TRUE"
```

Comment:
An inherently odd convention. Surely someone will suffer because of it.

Reply:
Yeah, didn't even notice. 😆 Only with the perfect input can you disable it.  Up, Up, Down, Down, Left, Right, Left, Right, B, A.